### PR TITLE
Update to Italian translation

### PR DESCRIPTION
**Pull request by MrOgeid**
New strings added for villager visits.
Some typos fixed, especially "Wedding season" not correctly translated in the Events section.

### CODE REVIEW

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -1103,3 +1104,8 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
 "All" = "Tutti";
 "Liked" = "Preferiti";
 "Residents" = "Residenti";
+
+// Added 2020-06-06
+"Villager visits" = “Parlare agli abitanti";
+"Who have you talked to today? Find the villagers you have visited and tap the home icon on the villager’s page to keep track." = “Con chi hai parlato oggi? Aggiungi qui i residenti con il pulsante della casa che trovi nella scheda di ogni abitante.”;
+"Long press on a villager to see more info about them" = “Tieni premuto su di un abitante per vedere la sua scheda.";
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:1111]:
```suggestion
"Villager visits" = "Parlare agli abitanti";
"Who have you talked to today? Find the villagers you have visited and tap the home icon on the villager’s page to keep track." = "Con chi hai parlato oggi? Aggiungi qui i residenti con il pulsante della casa che trovi nella scheda di ogni abitante.";
"Long press on a villager to see more info about them" = "Tieni premuto su di un abitante per vedere la sua scheda.";
```

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -763,7 +764,7 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
 "Trash Island" = "Isola Rifiuti";
 "Fins Island" = "Isola Pinne";
 "Falls Island" = "Isola Cascate";
-"Gold Island" = "Isola Oro";
+"Gold Island" = "Isola Minerali d’oro;
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:767]:
```suggestion
"Gold Island" = "Isola Minerali d’oro";
```

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 3]
```diff
@@ -709,8 +710,8 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
 
 
 // Added 2020-05-13
-"See more" = "Più info";
-"See less" = "Meno info";
+"See more" = “Mostra di più”;
+"See less" = “Mostra di meno”;
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:714]:
```suggestion
"See more" = "Mostra di più";
"See less" = "Mostra di meno";
```

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 4]
```diff
@@ -29,6 +29,7 @@
 // TodaySectionEditView
 "Drag & Drop to Rearrange" = "Trascina e rilascia per riordinare";
 "Hidden" = "Nascosto";
+“Subscribe” = “Abbonamento”;
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:32]:
```suggestion
"Subscribe" = "Abbonamento";
```

**Comment by TheVaan** [comment]
Updated quotes (you may should use an other editor - quotes were wrong).
I'll merge this. I saw that there are some keys duplicated. I don't know which one exactly  but if my excel table was right there were about 40 strings duplicated (e.g. found wedding season and able sisters twice). To be honest I didn't find a way how we can find these string.

**Comment by MrOgeid** [comment]
Thanks @TheVaan. I really need to try a new text editor that correctly handles quotes and line breaks.😅 I'll try Atom.
I noticed that the first file I translated when I started the Italian file had some duplicated strings. I thought that was needed for  different sections. But I can try to track them down to clean up.

**Comment by TheVaan** [comment]
@MrOgeid Yes, this wasn't your fault - it was mine :D I removed the duplicated string a few days ago from German and French translation but tbh I forgot to check Italian.
Atom should do it right. I also can recommend Visual Studio Code - it's nearly the same but I like git integration of VS Code more then the one in Atom. Feel free to choose!

**Pull request merged**